src/engine/source/indexerconnector/test/component/indexerConnector_test.cpp

```diff
@@ -266,27 +266,21 @@ TEST_F(IndexerConnectorTest, DISABLED_ConnectionWithSslCredentials)
  * the test coverage.
  *
  */
-TEST_F(IndexerConnectorTest, DISABLED_ConnectionWithCertsArray)
+TEST_F(IndexerConnectorTest, DISABLED_ConnectionWithCert)
```

Comment:
Should we enable this test?

Reply:
I have removed this test as it was testing that the merge is done correctly, but this functionality is no longer required.